mxcube3/routes/Login.py

```diff
@@ -125,7 +125,6 @@ def loginInfo():
 
     login_info = login_info["loginRes"] if login_info is not None else {}
     login_info = limsutils.convert_to_dict(login_info)
-    limsutils.update_mxcube_session(login_id, login_info)
```

Comment:
thanks, let me know if you have issues, this is a tricky part and different in our sites.